zsh-aws-vault.plugin.zsh

```diff
@@ -19,6 +20,48 @@ function avsh() {
   aws-vault exec $1 -- zsh
 }
 
+function avli() {
+  local login_url="$(avll $1)"
+
+  if _using_osx ; then
+    local browser="$(_find_browser)"
+
+    case $browser in
+      org.mozilla.firefox)
+        echo "${login_url}" | xargs /Applications/Firefox.app/Contents/MacOS/firefox --private-window
+        ;;
+      com.google.chrome)
+        echo "${login_url}" | xargs /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --incognito --new-window
+        ;;
+      *)
+        # NOTE PRs welcome to add your browser
+        echo "Sorry, I don't know how to launch your default browser ($browser) :-("
+        ;;
+    esac
+
+  else
+    # NOTE this is untested - PRs welcome to improve it.
+    echo "${login_url}" | xargs xdg-open
+  fi
+}
+
+function avp() {
```

Comment:
this is great - I just made the naming match up a bit better with the existing values